rtv/subscription_page.py

```diff
@@ -43,17 +44,49 @@ def refresh_content(self, order=None, name=None):
             self.nav = Navigator(self.content.get)
 
     @SubscriptionController.register(Command('PROMPT'))
-    def prompt_subreddit(self):
-        "Open a prompt to navigate to a different subreddit"
-
-        name = self.term.prompt_input('Enter page: /')
-        if name is not None:
-            with self.term.loader('Loading page'):
-                content = SubredditContent.from_name(
-                    self.reddit, name, self.term.loader)
-            if not self.term.loader.exception:
-                self.selected_subreddit = content
-                self.active = False
+    def add_subreddit(self):
+        "Open a prompt to add or remove multi/subreddit"
+
+        context = self.content.name
+        obj = self.get_selected_item()['object']
+
+        if context == 'My Subreddits':
+            name = self.term.prompt_input('Subscribe to: /')
+            if name is not None:
+                with self.term.loader('Adding /r/{}'.format(name, context)):
+                    for n in name.split('+'):
```

Comment:
Is supporting the `+` format really necessary here? I understand using it for initializing multireddits, but I can't imagine that somebody is going to want to subscribe to more than a single subreddit at once. The reason I bring it up is that the error handling could be a lot clearer (show a terminal notification if the subscription fails) if there is only one subreddit. I worry that silently ignoring invalid subreddits could be seen as an error from the end user's perspective.